package/src/frontend/documentHiddenState.js

```diff
@@ -0,0 +1,27 @@
+
+let isEventAdded = false;
+let callbacks = [];
+
+let muteInBackground = () => {  
+  callbacks.forEach((cb)=>{    
+    if (cb instanceof Function) {
```

Comment:
If we take this check out, developers could get more direct feedback if they register a non-function callback by mistake. Since we control the type of cb, I'd suggest taking out the `instanceOf` check here